wolfssl/openssl/ssl.h

```diff
@@ -137,8 +137,13 @@ typedef WOLFSSL_X509_VERIFY_PARAM X509_VERIFY_PARAM;
 #define CONF_get1_default_config_file   wolfSSL_CONF_get1_default_config_file
 typedef STACK_OF(ACCESS_DESCRIPTION) AUTHORITY_INFO_ACCESS;
 
+#ifdef WOLFSSL_QT
+#define CRYPTO_free                     wolfSSL_CRYPTO_free
```

Comment:
Should these be general to OPENSSL_EXTRA instead of WOLFSSL_QT only?

Reply:
Agree. Should be general to OPENSSL_EXTRA